src/main/kotlin/no/nav/dagpenger/rapportering/service/RapporteringService.kt

```diff
@@ -316,25 +316,21 @@ class RapporteringService(
         loginLevel: Int,
         headers: Headers,
     ): InnsendingResponse {
-        val lagretRapporteringsperiode = rapporteringRepository.hentRapporteringsperiode(rapporteringsperiode.id, ident)
-        if (lagretRapporteringsperiode != null && !lagretRapporteringsperiode.kanSendes) {
+        val kanSendes = rapporteringRepository.hentKanSendes(rapporteringsperiode.id)
```

Comment:
Ja, ok, da forutsetter vi at denne perioden finnes allerede i databasen. Men det er egentlig ok, fordi man måtte åpne denne periode for å starte innsending og vi henter periode fra Arena på det tidspunktet

Reply:
Hvis perioden ikke finnes vil `val kanSendes` bli satt til null. Vi behandler det videre som at den er false.